google_play_scraper/constants/element.py

```diff
@@ -174,3 +204,84 @@ class ElementSpecs:
         "developer": ElementSpec(None, [0,14]),
         "installs": ElementSpec(None, [0,15])
     }
+
+    DataSafety = {
+        "dataCollected": ElementSpec(
+            3,
+            [1, 2, 137, 4, 1, 0],
+            lambda collection: {
+                ElementSpec(None, [0, 1])
+                .extract_content(collection[i]): ElementSpec(
+                    None,
+                    [4],
+                    lambda entrys: [
+                        {
+                            "name": ElementSpec(None, [0]).extract_content(entrys[j]),
+                            "optional": ElementSpec(None, [1]).extract_content(
+                                entrys[j]
+                            ),
+                            "usage": ElementSpec(None, [2], None, None).extract_content(
+                                entrys[j]
+                            ),
+                        }
+                        for j in range(0, len(entrys))
+                    ],
```

Comment:
This block can be replaced to classic function. As you said, `dataCollected` and `dataShared` structure is simillar. It will resolve nesting of lambdas.

I like these element specification with one-line code. But complexity of data structure increases, we can change declaring extractor functions to classic function(using `def`).